app/views/_user_util_links.html.erb

```diff
@@ -28,13 +28,13 @@
     </li>
     <li class="osu-login">
       <%= link_to main_app.new_osu_session_path do %>
-        <span class="glyphicon glyphicon-log-in login-icon-only" aria-hidden="true"></span>
+        <span class="glyphicon glyphicon-log-in login-icon-only" aria-hidden="true"> <%= t('hyrax.toolbar.osu.login') %></span>
```

Comment:
I think we win a bit better at accessibility if we convert the sr-only below to be visible instead. Plus, it helps the odd font.
Just need to throw in a bit of CSS here for font color https://github.com/OregonDigital/OD2/blob/master/app/assets/stylesheets/oregon_digital/_home-page.scss#L46